opensearch/src/root/bulk.rs

```diff
@@ -179,6 +179,11 @@ where
         BulkCreateOperation::new(id, source)
     }
 
+    /// Creates a new instance of [bulk create operation](BulkCreateOperation) without explicit id
+    pub fn create_no_id(source: B) -> BulkCreateOperation<B> {
+        BulkCreateOperation::new_no_id(source)
```

Comment:
I'd prefer `without_id` over `no_id` for comprehensibility.
```suggestion
    /// Creates a new instance of [bulk create operation](BulkCreateOperation) without an explicit id
    pub fn create_without_id(source: B) -> BulkCreateOperation<B> {
        BulkCreateOperation::new_without_id(source)
```